### PR TITLE
SCALA-96 | Access Play! 2.0 configuration variables in application.conf

### PR DESCRIPTION
**Pull request by njacobs5074**
See http://jira.baeldung.com/browse/SCALA-96

### CODE REVIEW

**Comment by lor6** [comment]
retest this please

**Comment by lor6** [comment]
retest this please

**Comment by njacobs5074** [comment]
Hi Loredana. When I pull upstream master and try to build it locally, I get the following failure _on a fresh compile_
```
[info] *** 1 TEST FAILED ***
[error] Failed: Total 31, Failed 1, Errors 0, Passed 30
[error] Failed tests:
[error] 	com.baeldung.scala.akka.scheduler.TimerSpec
[error] (scala_akka / Test / test) sbt.TestsFailedException: Tests unsuccessful
```

However, if I re-run the tests a second time, it passes.

**Comment by njacobs5074** [comment]
OK, it built this time. Strange.

File: play-scala/configuration-access/app/controllers/HomeController.scala [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+import services.MyService
+
+/**
+  * This controller creates an `Action` to handle HTTP requests to the
+  * application's home page.
+  */
+@Singleton
+class HomeController @Inject() (
+    val myService: MyService,
+    val controllerComponents: ControllerComponents
```
Comment (KevinGilmore) [inline play-scala/configuration-access/app/controllers/HomeController.scala:14]: Use 2-space indent

File: play-scala/configuration-access/app/services/MyService.scala [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+      .map[Date](javax.xml.bind.DatatypeConverter.parseDateTime(_).getTime)
+  }
+}
+
+case class PlayerInfo(
+    name: String,
+    email: String,
+    age: Int,
+    signUpDate: Date,
+    twitterHandle: Option[String] = None
```
Comment (KevinGilmore) [inline play-scala/configuration-access/app/services/MyService.scala:23]: 2-space indent

File: play-scala/configuration-access/test/controllers/HomeControllerSpec.scala [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+  * Add your spec here.
+  * You can mock out a whole application including requests, plugins etc.
+  *
+  * For more information, see https://www.playframework.com/documentation/latest/ScalaTestingWithScalaTest
+  */
+class HomeControllerSpec
+    extends PlaySpec
+    with GuiceOneAppPerTest
+    with Injecting
+    with MockitoSugar {
```
Comment (KevinGilmore) [inline play-scala/configuration-access/test/controllers/HomeControllerSpec.scala:20]: 2-space indent

File: play-scala/configuration-access/test/services/MyServiceSpec.scala [hunk 4]
```diff
@@ -0,0 +1,50 @@
...
+import org.mockito.MockitoSugar
+import org.scalatestplus.play._
+import org.scalatestplus.play.guice._
+import play.api.test._
+
+class MyServiceSpec
+    extends PlaySpec
+    with GuiceOneAppPerTest
+    with Injecting
+    with MockitoSugar {
```
Comment (KevinGilmore) [inline play-scala/configuration-access/test/services/MyServiceSpec.scala:14]: 2-space indent

**Review by KevinGilmore — COMMENTED** [review]
A few places aren't using two-space indent.

File: play-scala/configuration-access/app/controllers/HomeController.scala (hunk 1 above)
Comment (njacobs5074) [reply play-scala/configuration-access/app/controllers/HomeController.scala]: Done

File: play-scala/configuration-access/app/services/MyService.scala (hunk 2 above)
Comment (njacobs5074) [reply play-scala/configuration-access/app/services/MyService.scala]: Done

File: play-scala/configuration-access/test/controllers/HomeControllerSpec.scala (hunk 3 above)
Comment (njacobs5074) [reply play-scala/configuration-access/test/controllers/HomeControllerSpec.scala]: Done

File: play-scala/configuration-access/test/services/MyServiceSpec.scala (hunk 4 above)
Comment (njacobs5074) [reply play-scala/configuration-access/test/services/MyServiceSpec.scala]: Done

**Review by glmartin — APPROVED** [review]

**Pull request merged**